setup.py

```diff
@@ -26,7 +26,8 @@
     'Programming Language :: Python :: 3.5',
     'Programming Language :: Python :: 3.6',
     'Programming Language :: Python :: Implementation',
-    'Programming Language :: Python :: Implementation :: CPython'
+    'Programming Language :: Python :: Implementation :: CPython',
+    'Framework :: ChaosToolkit :: plugin'
```

Comment:
Maybe  `Plugin` to match the case-senstive aspect of other components?

Reply:
Totally agreed, making those changes now.